.github/templates/build-packages.yml

```diff
@@ -87,8 +87,7 @@ jobs:
     steps:
       - #@ template.replace(checkoutCode())
       - #@ template.replace(setupAndroid())
-      - _: #@ template.replace(setupDotnet("6.0.417"))
-      - #@ setupWorkloads('tvos ios maccatalyst android')
+      - #@ template.replace(setupWorkloads('tvos ios maccatalyst android'))
```

Comment:
Are those changes part of this PR or something that got picked up from main? If the latter, can we also update the base PR so that the only changes left are the ones that need to be reviewed.

Reply:
Yes, this is coming from main, I'll merge that into the base PR

Reply:
Just done